Simplenote/Classes/SignupRemote.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+// MARK: - SignupRemote
+//
+class SignupRemote {
+    private let urlSession: URLSession
+
+    init(urlSession: URLSession = URLSession.shared) {
+        self.urlSession = urlSession
+    }
+
+    /// Send signup request for specified email address
+    ///
+    func signup(with email: String, completion: @escaping (_ success: Bool) -> Void) {
+        guard let request = request(with: email) else {
+            completion(false)
+            return
+        }
+
+        let dataTask = urlSession.dataTask(with: request) { (data, response, error) in
+            DispatchQueue.main.async {
+                // Check for 2xx status code
+                guard let response = response as? HTTPURLResponse, response.statusCode / 100 == 2 else {
+                    completion(false)
+                    return
+                }
+
+                completion(true)
+            }
+        }
+
+        dataTask.resume()
+    }
+
+    private func request(with email: String) -> URLRequest? {
+        guard let base64EncodedEmail = email.data(using: .utf8)?.base64EncodedString(),
+              let url = URL(string: SimplenoteConstants.signupURL) else {
+            return nil
+        }
+
+        var request = URLRequest(url: url,
+                                 cachePolicy: .reloadIgnoringLocalAndRemoteCacheData,
+                                 timeoutInterval: Constants.timeoutInterval)
+        request.httpMethod = Constants.httpMethod
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        request.httpBody = try? JSONEncoder().encode(["username": base64EncodedEmail])
```

Comment:
as noted in Slack, we did determine that sending the username inside a JSON body would be sufficient since JSON properly handles Unicode data without encoding/decoding issues. we can pass a JSON encoded object with a `username` field containing the the plain/raw email address.